examples/rsp-cra-18/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import './App.css';
+import {Provider, defaultTheme} from '@adobe/react-spectrum'
+import Lighting from './Lighting';
+import React from 'react'
+import BodyContent from './BodyContent';
+function App() {
+  let [selected, setSelection] = React.useState(false);
+
+  //changes the background color of the document based on the status of 'selected'
+  React.useEffect(() => {
```

Comment:
you may find this Codesandbox css of interest https://codesandbox.io/s/modern-cache-moxcxy?file=/src/styles.css
I think you're doing this so the full screen matches the provider background, but we can just make the provider take up the full screen